lib/api/objectGet.js

```diff
@@ -46,7 +48,8 @@ function objectGet(authInfo, request, log, callback) {
         const responseMetaHeaders = collectResponseHeaders(objMD, corsHeaders);
         // 0 bytes file
         if (objMD.location === null) {
-            if (request.headers.range) {
+            const range = request.headers.range;
+            if (range && validRangeOnEmptyFile(range)) {
```

Comment:
// aws returns the empty object if there is an invalid range but sends an invalidRange error
// if the range is valid on an empty object